### PR TITLE
Add works section

### PR DESCRIPTION
**Pull request by TahenyBELGUITH**
In this project I use:

1. Html box model

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,27 +1,35 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Fork Me! FCC: Test Suite Template</title>
```
Comment (TahenyBELGUITH) [inline index.html:6]: it's better to choose an appropriate title

File: index.html [hunk 2]
```diff
@@ -1,27 +1,35 @@
...
-            <ul><li><a href="#welcome-section" target="#"> Welcome </a></li></ul>
-            <ul><li><a href="#projects" target="#">projects</a></li></ul>
-      </nav>
-    <section id="welcome-section" class="welcome-section">
-      <h1>Hi, we are Taheny and Raphael <br> a web developers </h1>
-    </section>
-    <section id="projects" class="projects">
-      <h1>Here is our projects section</h1>
-      <div class="project-tile">
-        <a href=#></a>
```
Comment (TahenyBELGUITH) [inline index.html:21]: it's better t not use empty ancher tag

**Review by TahenyBELGUITH — COMMENTED** [review]
please check the comments below

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @TahenyBELGUITH, 

:+1: This is good to make it this far, your **PR Review** is really nice, congratulations on the good work you have done so far.
:+1: You have used Github inline comments nicely. :muscle: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️
 
2. I think it's better to always includes an explanation, a "why", for each requested change.

   * This is a well explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
   * This request lacks a "why": *Why don't you use an article tag here instead of a div?*

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

1. I think It's better to use a couple of emojis and/or gifs in the review.
   * They makes the code review more human-friendly :alien::robot:.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Temmarie — APPROVED** [review]
## Status: Approved ✔️ 
### Hi @TahenyBELGUITH  , 
#### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
#### Congratulations! 🎉 

#### Cheers and Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**